api/v1/helpers/verbs/utils.js

```diff
@@ -791,10 +792,16 @@ function responsify(rec, props, method) {
   return o;
 } // responsify
 
+function getHash(string) {
+  return md5(string);
+}
```

Comment:
can we add some prefix like 'refocusCache_hash(key)' ?

Reply:
what's the benefit of adding prefix like that?

Reply:
I was thinking there's nothing specific that makes this 'refocus cache', and that anytime someone needs a hash of a string it can be used (hence a more generic name). What do you think?